app/build.gradle

```diff
@@ -246,8 +246,6 @@ dependencies {
     implementation "com.github.lisawray.groupie:groupie:${groupieVersion}"
     implementation "com.github.lisawray.groupie:groupie-viewbinding:${groupieVersion}"
 
-    // Circular ImageView
-    implementation "de.hdodenhof:circleimageview:3.1.0"
```

Comment:
How much the APK size is decreased with the library removal?

Reply:
Debug builds are around 5KB smaller.

Release builds are around 1KB larger.

Nothing large 👍